### PR TITLE
🐛 add github token to be able to download artifacts before publishing to pypi

### PR DESCRIPTION
**Pull request by bisgaard-itis**

### CODE REVIEW

File: .github/workflows/publish-python-client.yml [hunk 1]
```diff
@@ -19,6 +19,8 @@ jobs:
     steps:
       - name: Download wheels
         run: gh run download ${{ github.event.workflow_run.id }}
+        env:
+          GH_TOKEN: ${{ github.token }}
```
Comment (wvangeit) [inline .github/workflows/publish-python-client.yml:23]: sorry, i never used this particular 'github.token'. What is this token? Is this the API token running the job? (I'm just wondering if there is any danger exposing this in the environment variables/logs.

Comment (wvangeit) [reply .github/workflows/publish-python-client.yml:23]: Ah ok, it seems to be a 1-time token? Seems to be ok then.

**Review by wvangeit — APPROVED** [review]

File: .github/workflows/publish-python-client.yml (hunk 1 above)
Comment (bisgaard-itis) [reply .github/workflows/publish-python-client.yml:24]: It is a good point. I added a label in the workflow which ensures the token is only granted read access which should be fine. Here it is only used to download the artifacts from the build workflow

Comment (wvangeit) [reply .github/workflows/publish-python-client.yml:24]: Yep, very good.

**Pull request merged**